File: 2206-DivideArrayIntoEqualPairs/2206-DivideArrayIntoEqualPairs.py
# Last updated: 9/25/2026, 2:32:25 PM
class Solution(object):
    def divideArray(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        l=set(nums)
        for i in l:
            if nums.count(i)%2!=0:
                return False
        return True

File: 2206-DivideArrayIntoEqualPairs/2206-DivideArrayIntoEqualPairs.java
// Last updated: 9/25/2026, 2:32:21 PM
class Solution {
    public boolean divideArray(int[] nums) {
        Arrays.sort(nums);
        for(int i=0;i<nums.length;i+=2){
            if(nums[i]!=nums[i+1]){
                return false;
            }
        }
        return true;
    }
}